src/content/docs/zh-cn/features/websocket.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Websocket
+description: 在 JavaScript 中使用 Rust 客户端打开 WebSocket 连接。
+---
+
+import PluginLinks from '@components/PluginLinks.astro';
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+<PluginLinks plugin="websocket" />
+
+在 JavaScript 中使用 Rust 客户端打开 WebSocket 连接。
+
+## 支持的平台
+
+- Windows
+- Linux
+- macOS
+
+## 设置
+
+_这个插件要求 Rust 版本至少是 **1.75**_
+
+首先安装 websocket 插件。
```

Comment:
```suggestion
首先安装 WebSocket 插件。
```
Starting to think maybe we should capitalize all English plugin names, but let's just change this one for now for consistency's sake.